src/winforms/toga_winforms/window.py

```diff
@@ -36,16 +37,72 @@ def create(self):
         self.toolbar_items = None
 
     def create_toolbar(self):
-        self.toolbar_native = WinForms.ToolStrip()
+        self.toolbar_native = WinForms.MenuStrip()
         for cmd in self.interface.toolbar:
             if cmd == GROUP_BREAK:
                 item = WinForms.ToolStripSeparator()
             elif cmd == SECTION_BREAK:
                 item = WinForms.ToolStripSeparator()
             else:
-                item = WinForms.ToolStripButton()
+                cmd.native = cmd.bind(self.interface.factory)
+                native_icon = cmd.icon.bind(self.interface.factory).native
+                if cmd.icon is not None:
+                    item = WinForms.ToolStripMenuItem(cmd.label, native_icon.ToBitmap())
+                else:
+                    item = WinForms.ToolStripMenuItem(cmd.label)
+
+                def add_handler(cmd):
+                    action = cmd.action
+
+                    def handler(sender, event):
+                        return action(None)
+
+                    return handler
+
+                item.Click += add_handler(cmd)
             self.toolbar_native.Items.Add(item)
 
+    def create_menus(self):
+
+        toga.Group.FILE.order = 0
+
+        def add_handler(cmd):
+            action = cmd.action
+
+            def handler(sender, event):
+                return action(None)
+
+            return handler
```

Comment:
This looks like it's repeating code in `create_toolbar` - can we pull this out as a general utility method that both toolbars and menus can use?

Reply:
Is this the simplification you mentioned in the comment? I really have a problem with git... What should I have done to merge all the changes to this branch in my fork?

Is winforms/libs a good place to put this kind of code (add_handler function)?